Common/src/main/java/com/blamejared/crafttweaker/api/plugin/IJavaNativeIntegrationRegistrationHandler.java

```diff
@@ -4,14 +4,80 @@
 import com.blamejared.crafttweaker.api.zencode.IPreprocessor;
 import com.blamejared.crafttweaker.api.zencode.ZenTypeInfo;
 
+/**
+ * Handles registration of the various components required to integrate ZenCode scripts and Java code.
+ *
+ * <p>The various integration components are made up of native classes, zen classes, globals, and preprocessors. Refer
+ * to each specific method for more information.</p>
+ *
+ * @since 9.1.0
+ */
 public interface IJavaNativeIntegrationRegistrationHandler {
     
+    /**
+     * Registers the specified class as a native type for ZenCode.
+     *
+     * <p>A native type is a class that belongs that has not been written with ZenCode scripts in mind, but that is
```

Comment:
```suggestion
     * <p>A native type is a pre-existing class that has not been written with ZenCode scripts in mind, but that is
```

Reply:
What the hell was I thinking when I wrote this? :joy: What even is a *class that belongs*?